doc/content/getting-started/installation/running-the-stack.md

```diff
@@ -65,7 +67,7 @@ $ docker-compose run --rm stack is-db create-oauth-client \
   --id console \
   --name "Console" \
   --owner admin \
-  --secret the secret you generated before \
+  --secret     # the secret from ttn-lw-stack-docker.yml you generated before \
```

Comment:
I'd recommend putting it between angle brackets instead of a comment. Bash doesn't really like this style and in case they just fill it before the `#` the command will end there and the remaining parameters will be ignored.

Source: https://unix.stackexchange.com/a/19126